Created maureen-gamradt-advice.html branch, created maureen-gamradt-a…

Gamradical opened the pull request:
…dvice.css, created link to maureen-gamradt-advice.html on index.html

redrambles commented:
Hey @Gamradical !

Nice work on this!! I see you've added a `br` tag for the student link that precedes yours - thank you! But somewhere in the process of editing the index.html file, the closing `</section>` tag was lost, [see here.](https://github.com/skillcrush/git-practice/pull/579/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L504)

Can you add that back in?

Once you do - save the file, add and commit the change and then push up the `advice` branch once more. Then we should be all set to merge in your code.

Let us know if you have any questions!

Cheers,
Ann :)

Gamradical commented:
Oh golly!
So, I went back and fixed it, I think.
My steps were as follows:
Navigated to git-practice in Terminal.
open . to access the index.html.
Opened  index.html in Atom.
Added closing section tag.
Saved changes.
In terminal, I added the changed file using git add -A.
Then checked the status in Terminal.
git commit -m with my commit message.
Finally, git push origin avice.

I hope that was right!?!  I am still pretty nervous about messing all this
up.....
Let me know what you think, and if I need to do something else.
All the best,
Maureen

On Fri, Apr 15, 2022 at 10:07 AM Ann Cascarano ***@***.***>
wrote:

> Hey @Gamradical <https://github.com/Gamradical> !
>
> Nice work on this!! I see you've added a br tag for the student link that
> precedes yours - thank you! But somewhere in the process of editing the
> index.html file, the closing </section> tag was lost, see here.
> <https://github.com/skillcrush/git-practice/pull/579/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L504>
>
> Can you add that back in?
>
> Once you do - save the file, add and commit the change and then push up
> the advice branch once more. Then we should be all set to merge in your
> code.
>
> Let us know if you have any questions!
>
> Cheers,
> Ann :)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/skillcrush/git-practice/pull/579#issuecomment-1100164155>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASASOPMGGKT27HJTPE3SPN3VFGAZZANCNFSM5TQTENSQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

twosavoie commented:
Hey @Gamradical,

Great job! Your code is all [merged in](https://media.tenor.com/images/4a6158ad4f30c71f4fecf224cec12eb1/tenor.gif).

xoxo,
Lisa
